prometheus-to-sd/translator/translator.go

```diff
@@ -143,6 +143,11 @@ func translateOne(config *config.CommonConfig,
 		EndTime: time.Now().UTC().Format(time.RFC3339),
 	}
 	metricKind := extractMetricKind(mType)
+	// Prometheus Cumulative -> Stackdriver Gauge translation
+	if metricKind == "GAUGE" && isMetricGaugeToCumulativeWhitelisted(name, config) {
+		metricKind = "CUMULATIVE"
+		interval.StartTime = start.UTC().Format(time.RFC3339)
```

Comment:
This is not needed, it's going to be processed by branch below.

Reply:
Done